src/main/scala/org/constellation/gossip/validation/MessageValidator.scala

```diff
@@ -2,14 +2,15 @@ package org.constellation.gossip.validation
 
 import cats.data.Validated
 import cats.implicits._
+import com.typesafe.scalalogging.StrictLogging
 import org.constellation.gossip.state.GossipMessage
 import org.constellation.schema.Id
 import org.constellation.schema.signature.HashSignature
 
 import scala.annotation.tailrec
-import scala.util.Try
+import scala.util.{Failure, Success, Try}
 
-class MessageValidator(selfId: Id) {
+class MessageValidator(selfId: Id) extends StrictLogging {
```

Comment:
I'd suggest avoiding using StrictLogging. Instead of that, if validation needs to make some side effects, it should be wrapped into the monad (IO)

Reply:
Ok